ffwd-client/src/main/java/com/spotify/ffwd/v1/Value.java

```diff
@@ -40,14 +40,14 @@ public static Value doubleValue(double value) {
     return DoubleValue.create(value);
   }
 
-  public static Value distributionValue(ByteBuffer byteBuffer) {
-    return DistributionValue.create(byteBuffer);
+  public static Value distributionValue(ByteString byteString) {
```

Comment:
This would be a breaking change, which I think we can live with if we are sure no one is depending on this.
Otherwise, we would need to keep the old method signature too

Reply:
No one is using it. Nothing will break.